app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :body, presence: true, length: { maximum: 1000 }
+  validates :user_id, presence: true
```

Comment:
- [ ] The belongs_to relation already validates the user_id for you, so this line is unnecessary.

- [ ] You are missing the title validation, the post should have both a body and a title.